Schedule page

kamcknig opened the pull request:
Initial work simply to route to a page, get the event listings and simply display their names in divs for now.

kamcknig committed:
Added the schedule page component and started the module for the events API

kyle-mcknight committed:
added prettier rule for end of line to prevent warnings on windows machines

kyle-mcknight committed:
Retrieved list of events and displayed their names

kyle-mcknight committed:
Updated comment for time format

scriptist commented on package.json:
```diff
@@ -50,7 +50,12 @@
+      "prettier/prettier": [
+        "warn",
+        {
+          "endOfLine": "auto"
```
Most git repos only end lines with a line feed - is `auto` necessary for your dev environment? Is there any way we could get away with just line feed?

kamcknig replied:
I'll have to look into the settings of my IDE, the only way I could get it to stop was to set its line endings to LF but at the time since I had never tried, I only saw a way to do it per file. There is probably a way to do it per project.

scriptist commented on src/api/TwitarrAPI3Events.ts:
```diff
@@ -0,0 +1,29 @@
+}
+
+export default class TwitarrAPI3Events extends TwitarrAPI3Module {
+  getEvents = this.createAPIMethod<null, EventData[]>({
```
If you want this to have no params, you can use `void` here.

```suggestion
  getEvents = this.createAPIMethod<void, EventData[]>({
```

kamcknig replied:
Oh yeah, duh. Thanks

scriptist reviewed (changes requested):
Fantastic work!

kamcknig committed:
Addressing comments from PR. Using void instead of null for request configuration. Removing line-endings config from package.json

kamcknig committed:
Updated schedule page

- Refactored the event data display into a separate functional component
- displayed title and location of event. If the event is an
  "Official"event type then add the wheel logo to the title

scriptist reviewed (approved):
🎉
